ui/app/AppLayouts/Profile/views/wallet/MainView.qml

```diff
@@ -36,7 +37,18 @@ Column {
     StatusSettingsLineButton {
         text: qsTr("Manage Assets & List")
         height: 64
-        onClicked: goToNetworksView()
+        onClicked: Global.openPopup(tokenSettingsModalComponent)
+    }
+
+
+    Component {
+        id: tokenSettingsModalComponent
+        TokenSettingsModal {
+            walletStore: root.walletStore
+            onClosed: {
```

Comment:
I have been doing this but I dont think it is needed at all. Just writing it here cause it came to mind, I will check if this needs to be removed everywhere, later on. Ignore it

Reply:
Not sure if you want me to remove it or you will check and remove globally :confused: